src/test/java/de/retest/web/WebElementPeerTest.java

```diff
@@ -44,4 +44,9 @@ void filled_web_data_should_result_in_outline_with_corresponding_values() throws
 		assertThat( rectangle.getHeight() ).isEqualTo( 4.0 );
 	}
 
+	@Test
+	void normalize_should_remove_apostrophe() {
```

Comment:
Theoretical what if `""`, `"\""`, `"    \""`, `"\"   "`.

Reply:
Blanks should be trimmed. But the other cases ... would wait for them to occur in practice, to have a better understanding.